### DISCUSSION TITLE
Request: Support Darwinia2 with snapshot.

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#7] [General]
So we can have the possibilities of moving dcdao to Darwinia2.

Currently the gas price on Ethereum mainet is going high, do we need consider to move dcdao to L2 such as Arbitrum/ZKSync first?

**Comment by hackfisher** [comment]
@furoxr you said we submit a request to snapshot, is it a pull request or form?

**Comment by furoxr** [comment]
Yes, it was merged 2 days before, https://github.com/snapshot-labs/snapshot.js/pull/825. I think they have not deployed yet

**Comment by hackfisher** [comment]
https://darwinianetwork.medium.com/harnessing-the-power-of-snapshot-and-darwinia-chain-building-a-dao-on-darwinia-chain-29253597995a
